kimetaro.py

```diff
@@ -7,6 +7,9 @@
 import discord
 import pysnooper
 
+# Value initialization
+CALLCOMMAND_SUFFIX = ''
```

Comment:
The name `CALLCOMMAND_SUFFIX` is not good.
(Two words are combined)

How about `COMMAND_SUFFIX` or `MESSAGE_SUFFIX` ?
As indicated by the `message.content.startswith` method, in discord.py `CALLCOMMAND` stands for message.

Reply:
sed -i '' s/CALLCOMMAND_SUFFIX/COMMAND_SUFFIX/g kimetaro.py